tests/test-sessions/test-sessions-common/src/main/resources/ProxyableFactory.java

```diff
@@ -0,0 +1,34 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2019 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+import java.lang.ClassLoader;
+import java.lang.reflect.Proxy;
+
+public class ProxyableFactory
+{
+    public static Proxyable newProxyable (ClassLoader cl)
+    {
+        return getProxy(cl, new ProxyableInvocationHandler());
+    }
+
+    public static Proxyable getProxy(ClassLoader classloader, ProxyableInvocationHandler pih)
```

Comment:
I would call this method `newProxy()` rather than `getProxy()` to match with the overloaded one above.